scripts/tizen/4.0/packaging/caffe2.spec

```diff
@@ -0,0 +1,180 @@
+Name:           caffe2
+Version:        0.7.0
+Release:        0
+Summary:        A deep learning, cross platform ML framework with the flexibility of Python and the speed of C++.
+Group:          Development/Tools
+License:        Apache-2.0
+URL:            https://github.com/caffe2/caffe2
+Distribution:   Tizen
+Vendor:         Samsung Electronics
+Packager:       Geunsik Lim <geunsik.lim@samsung.com>
+Source0:        %{name}-%{version}.tar.gz
+Source1001:     %{name}.manifest
+Patch0:         %{name}-eigen3-path.patch
+
+BuildRequires:  make cmake gcc binutils glibc glibc-devel cpp libstdc++
+BuildRequires:  protobuf-devel
+BuildRequires:  eigen3-devel
+BuildRequires:  openblas-devel
+BuildRequires:  lmdb-devel
+BuildRequires:  leveldb-devel
+BuildRequires:  gflags-devel
+BuildRequires:  boost-devel
+BuildRequires:  opencv-devel mesa
+
+BuildRequires:  python-python-gflags
+BuildRequires:  python = 2.7
+BuildRequires:  python-numpy-devel >= 1.7
+BuildRequires:  python-pybind11-devel
+
+Requires:       libleveldb
+Requires:       libopenblas0
+Requires:       gflags
+Requires:       glog
+Requires:       boost
+Requires:       python-pybind11
+Requires:       opencv
+
+# Note that do not add git package to keep the build structure of GBS/OBS
+# BuildRequires:  git
+
+%description
+Caffe2 is a deep learning framework that provides an easy and straightforward
+way for you to experiment with deep learning and leverage community contributions
+of new models and algorithms. You can bring your creations to scale using the
+power of GPUs in the cloud or to the masses on mobile with Caffe2's cross-platform
+libraries.
+
+%package devel
+Summary:	caffe2 development package
+Group:		Development/Libraries/C and C++
+Provides:	caffe-devel = %{version}-%{release}
+Requires:	%{name} = %{version}
+Requires:	glibc-devel
+Requires:	gflags-devel
+Requires:	openblas-devel
+Requires:	protobuf-devel
+Requires:	glog-devel
```

Comment:
Don't many of these support pkgconfig?

Reply:
Actually, some packages support `pkg-config` to provides a unified interface. If needed in the future for more effective development, it will be considered consistently to use `pkgconfig(...)`.